models.py

```diff
@@ -1,14 +1,25 @@
 from comandos_bd import insert,select, delete,update
-def insert_filme(nome_filme,ano):
-    insert('filmes',['nome_filme','ano'],[nome_filme,ano])
+
 def insert_diretor(nome_completo):
     insert('diretores',['nome_completo'],[nome_completo])
+
 def get_diretor(nome_completo):
     select('diretores','nome_completo',nome_completo)
+
 def delete_dir(id):
     delete('diretores','id',id)
+
 def alter_diretor(id, nome_completo):
     update('diretores','id',id,['nome_completo'],[nome_completo])
 
 def select_diretor(nome_completo):
     select('diretores','nome_completo', nome_completo)
+
+def insert_genero(nome):
+    insert('generos',['nome'],[nome])
+
+def get_genero(nome):
+    select('generos','nome',nome)
```

Comment:
vamos mudar esse para:
```suggestion
def get_genero(id):
    return select('generos','id',id)
```
pra fazermos essas consultas de get usando o id que o insert retorna